web/modules/custom/joinup_core/src/EventSubscriber/GroupManageContentSubscriber.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\joinup_core\EventSubscriber;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Alters the 'view.group_content_management.manage' route.
+ */
+class GroupManageContentSubscriber extends RouteSubscriberBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function alterRoutes(RouteCollection $collection) {
+    if ($route = $collection->get('view.group_content_management.manage')) {
+      $route->setOption('_admin_route', TRUE);
```

Comment:
Here the overview is declared as an admin page but the moderators do not have access to the admin theme. Do you think this would be worthwhile to add? Perhaps in a separate issue?

Reply:
@pfrenssen Generally I think it's a good idea. Opened https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-4045

Reply:
About making admin path a View path, I just realised that I've opened a core ticket long time ago, in Drupal DevDays Milan https://www.drupal.org/node/2719797. That would be the correct way.

Reply:
@claudiu-cristea just mentioned this issue which is relevant: https://www.drupal.org/node/2719797